Packs/Zscaler/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Zscaler",
     "description": "Zscaler is a cloud security solution built for performance and flexible scalability.",
     "support": "xsoar",
-    "currentVersion": "1.0.5",
+    "currentVersion": "1.0.6",
```

flatter yourself  :)
```suggestion
    "currentVersion": "1.1.0",
```

---

:)